tests/framework/installer/ceph_manifests.go

```diff
@@ -1051,6 +1051,7 @@ rules:
   - batch
   resources:
   - jobs
+	- cronjobs
```

Comment:
indent looks off.

Reply:
I see CI is failing due to indent, in `manifests.go` we use `space` instead of `tab` IIRC and earlier I had  similar kind of issue. Not sure but this might be the reason.